### PR TITLE
Add tooltip with escape controls

### PR DESCRIPTION
**Pull request by Valar05**
Add accessible tooltip using reactstrap tooltip with escape to close and sectable text.

### CODE REVIEW

File: packages/tooltip/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@availity/tooltip",
+  "version": "2.1.2",
```
Comment (jordan-a-young) [inline packages/tooltip/package.json]: Can you change this to 1.0.0 please?

File: packages/tooltip/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "bugs": {
+    "url": "https://github.com/Availity/availity-react/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Availity/availity-react.git",
+    "directory": "packages/tooltip"
+  },
+  "license": "MIT",
+  "author": "Kyle Gray <kyle.gray@availity.com>",
```
Comment (jordan-a-young) [inline packages/tooltip/package.json]: You can change this to your own info or mine if you don't want to put your own

File: packages/tooltip/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "build": "tsup src/index.ts --format esm,cjs --dts",
+    "dev": "tsup src/index.ts --format esm,cjs --watch --dts",
+    "clean": "rm -rf dist",
+    "clean:nm": "rm -rf node_modules",
+    "bundlesize": "bundlesize",
+    "publish": "yarn npm publish --tolerate-republish --access public",
+    "publish:canary": "yarn npm publish --access public --tag canary"
+  },
+  "dependencies": {
+    "@availity/icon": "workspace:*",
```
Comment (jordan-a-young) [inline packages/tooltip/package.json]: We should be able to remove this dependency as well as @availity/message-core

**Review by jordan-a-young — CHANGES_REQUESTED** [review]

File: packages/help/tsconfig.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["."],
+  "include": [".", "src/Tooltip.stories.tsx"],
```
Comment (LauRoxx) [inline packages/help/tsconfig.json]: looks like this is leftover from when we were going to put it in the help package

File: packages/tooltip/package.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  "devDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "reactstrap": "^8.10.1",
+    "tsup": "^5.12.7",
+    "typescript": "^4.6.4"
+  },
+  "peerDependencies": {
+    "react": ">=16.3.0"
```
Comment (LauRoxx) [inline packages/tooltip/package.json]: add reactstrap here

**Review by jordan-a-young — APPROVED** [review]

**Pull request merged**